hack/update-api-reference-docs.sh

```diff
@@ -71,6 +71,7 @@ for ver in $VERSIONS; do
   docker run ${user_flags} \
     --rm -v "${TMP_IN_HOST}":/output:z \
     -v "${SWAGGER_PATH}":/swagger-source:z \
+    --net=host -e "https_proxy=${KUBERNETES_HTTPS_PROXY}" \
```

Comment:
Thanks, @ixdy. 
I test the two ways you suggested. `${KUBERNETES_HTTPS_PROXY:-}` is OK. But I get an error `flag provided but not defined: -e https_proxy` in the `${KUBERNETES_HTTPS_PROXY:+"-e https_proxy='${KUBERNETES_HTTPS_PROXY}'"}` way. I'm not sure why. 
I commit the first way, since it's effective and more readable. PTAL. If it's OK, do I need to squash the commits?